edb/server/bootstrap.py

```diff
@@ -728,7 +728,7 @@ async def _make_stdlib(
         delta_command = s_ddl.delta_from_ddl(
             ddl_cmd, modaliases={}, schema=schema, stdmode=True)
 
-        if debug.flags.delta_plan_input:
+        if debug.flags.delta_plan:
```

Comment:
This should just be dropped, because `_process_delta` already dumps things, so you'll just be dumping the same thing twice.